### PR TITLE
Fixing ReadMe Issues

### PR DESCRIPTION
**Pull request by elamaran11**
Fixing ReadMe Issues

### CODE REVIEW

**Review by csantanapr — APPROVED** [review]
Thanks for the fix!

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Follow the following steps to get started.
     ```bash
     aws route53 list-hosted-zones-by-name --dns-name <YOUR_DOMAIN_NAME> --query 'HostedZones[0].Id' --output text | cut -d'/' -f3
     # in the setups/config file, update the zone id.
-    HOSTEDZONE_ID=ZO020111111
+    HOSTEDZONE_ID: ZO020111111
```
Comment (nimakaviani) [inline README.md:122]: this should actually be `hosted_zone_id: ` if we want to get this right. can you update the PR plz?

**Review by nimakaviani — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (elamaran11) [reply README.md]: @nimakaviani Yes my bad, fixed it, Please merge this. Thanks!

**Review by nimakaviani — APPROVED** [review]
LGTM

**Pull request merged**